src/equisolve/numpy/models/linear_model.py

```diff
@@ -299,6 +291,9 @@ def fit(
         self._validate_data(X, y)
         self._validate_params(X, alpha, sample_weight)
 
+        # Remove all gradients here. This is a workaround for now....
+        alpha = equistore.remove_gradients(alpha)
+
```

Comment:
I have to do some work on `equistore.equal_metadata` before removing this.

The current status of `equistore.equal_metadata` is that it checks "value" blocks and gradient "blocks". However, here we only want to check the "value" blocks of `alpha` and `X`. `X` can have gradients but `alpha` should not have any.

Reply:
I see, maybe you can comment this workaround a bit so we can already merge it
```suggestion
        # Remove all gradients here. This is a workaround until we can exclude gradients for metadata check (IDEALLY linking to some issue in equistore)
        alpha = equistore.remove_gradients(alpha)

```